spec/features/collection_drafts/forms/additional_attribute_type_spec.rb

```diff
@@ -57,6 +55,7 @@
         select 'Integer', from: 'Data Type'
         fill_in 'Parameter Range Begin', with: '5'
         fill_in 'Parameter Range End', with: '5'
+        find('body').click
```

Comment:
Was clicking the body not part of the last test?

Reply:
As I mentioned, previously using `fill_in` would also trigger a blur, which would allow validation or other JS activity we need to occur. Somehow with the change over, that is not happening so I need to click on something to trigger the blur.